ionoscloud/resource_dbaas_mariadb_cluster.go

```diff
@@ -37,7 +38,7 @@ func resourceDBaaSMariaDBCluster() *schema.Resource {
 				Description:      "The cluster location",
 				Optional:         true,
 				ForceNew:         true,
-				ValidateDiagFunc: validation.ToDiagFunc(validation.StringInSlice(constant.MariaDBClusterLocations, false)),
+				ValidateDiagFunc: validation.ToDiagFunc(validation.StringInSlice(constant.DBaaSClusterLocations, false)),
```

Comment:
probably change this to the new constant name

Reply:
@cbeti-ionos done